### PR TITLE
Generate local-kyma-installer.yaml and cluster-kyma-installer.yaml

### PR DESCRIPTION
**Pull request by kfurgol**
**Description**

Changes proposed in this pull request:

- add script that generates local-kyma-installer.yaml or cluster-kyma-installer.yaml 

**Related issue(s)**
Create release branch and pre release in github #867

### CODE REVIEW

File: installation/scripts/append-configmap-from-versions-env-to-installer-yaml.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+if [ ! -f ${INSTALLER_YAML_PATH} ]; then
+    echo "${INSTALLER_YAML_PATH} not found"
+    exit 1
+fi
+
+TMPFILE=$(mktemp ${KYMA_PATH}/installation/scripts/temp.XXXXXX)
+
+echo "---" >> ${TMPFILE}
+kubectl create cm versions --from-env-file ${VERSIONS_ENV_PATH} -n kyma-installer --dry-run -o yaml >> ${TMPFILE}
+echo "  labels:" >> ${TMPFILE}
```
Comment (Tomasz-Smelcerz-SAP) [inline installation/scripts/append-configmap-from-versions-env-to-installer-yaml.sh]: This relies on "metadata" section being last in the file. This seems to be ture currently, but is not granted forever. Please find a way to inject it into "metadata" section reliably.

Comment (strekm) [reply installation/scripts/append-configmap-from-versions-env-to-installer-yaml.sh]: for now i would assume that metadata section is the last one and address it in separate issue

File: installation/scripts/generate-kyma-installer.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+KYMA_PATH="${CURRENT_DIR}/../.."
+INSTALLER_YAML_PATH="${KYMA_PATH}/installation/resources/installer.yaml"
+UI_TEST_SECRET_PATH="${KYMA_PATH}/installation/resources/ui-test-secret.yaml.tpl"
+
+VERSIONS_ENV_PATH=""
+INSTALLER_CONFIG=""
+OUTPUT_PATH=""
+
+if [[ ! $# -eq 2 ]] ; then
+    echo 'Provide correct input arguments'
```
Comment (strekm) [inline installation/scripts/generate-kyma-installer.sh]: what are correct arguments?

File: installation/scripts/generate-kyma-installer.sh [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    exit 1
+fi
+
+case "$1" in
+    "local")   
+        INSTALLER_CONFIG="${KYMA_PATH}/installation/resources/installer-config-local.yaml.tpl"
```
Comment (strekm) [inline installation/scripts/generate-kyma-installer.sh]: i'm not sure wether we want to have the file hardcoded. Having path to the file as parameter would be more flexible i guess but then condition about ui test secret might not be possible

File: installation/scripts/generate-kyma-installer.sh [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+if [ ! -f ${INSTALLER_YAML_PATH} ]; then
+    echo "${INSTALLER_YAML_PATH} not found"
+    exit 1
+fi
+
+TMPDIR=`mktemp -d "${KYMA_PATH}/temp-XXXXXXXXXX"`
+CR_PATH="${TMPDIR}/installer-cr-local.yaml"
+bash ${KYMA_PATH}/installation/scripts/create-cr.sh --url "" --output "${CR_PATH}" --version 0.0.1
+
+#TODO should be fixed with https://github.com/kyma-project/kyma/issues/959
```
Comment (strekm) [inline installation/scripts/generate-kyma-installer.sh]: move this to do near relevant section ;) and maybe you can be more specific (something like this approach base on metadata section being the last one which might not be the case  with different version of kubectl )

File: installation/scripts/generate-kyma-installer.sh [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+cat ${INSTALLER_CONFIG} >> ${OUTPUT_PATH}
+
+echo "---" >> ${OUTPUT_PATH}
+
+cat ${CR_PATH} >> ${OUTPUT_PATH}
+
+echo "---" >> ${OUTPUT_PATH}
+
+if [ "$1" == "cluster" ]; then
```
Comment (strekm) [inline installation/scripts/generate-kyma-installer.sh]: can't this file be par of installer-config-cluster.yaml.tpl? please check that and change if possible -> one `cat`, one `if` and one file less. also problem with hardcoded config path might be resolved

File: installation/scripts/generate-kyma-installer.sh [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+case "$1" in
+    "local")   
+        INSTALLER_CONFIG="${KYMA_PATH}/installation/resources/installer-config-local.yaml.tpl"
+        OUTPUT_PATH="${KYMA_PATH}/kyma-installer/local-kyma-installer.yaml" ;;
+    "cluster") 
+        INSTALLER_CONFIG="${KYMA_PATH}/installation/resources/installer-config-cluster.yaml.tpl"
+        OUTPUT_PATH="${KYMA_PATH}/kyma-installer/cluster-kyma-installer.yaml" ;;    
+    *) 
+        echo 'Provide which installer you want to generate by passing as argument "local" or "cluster"'
```
Comment (Tomasz-Smelcerz-SAP) [inline installation/scripts/generate-kyma-installer.sh]: Add "usage" function that explains usage and returns error code 1. Refer to this function anytime args are incorrect.

**Pull request merged**